main.py

```diff
@@ -2,6 +2,53 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    actors = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
+        ("Will", "Smith"), ("Jaden", "Smith"), ("Scarlett", "Johansson")
+    ]
+
+    genres = ["Western", "Action", "Dramma"]
+
+    for names in actors:
+        Actor.objects.create(
+            first_name=names[0],
+            last_name=names[1]
+        )
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    Genre.objects.filter(
+        name="Dramma",
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        last_name="Klooney"
```

Comment:
Task says that we have to update surname for George Klooney. If there was another guy with the same surname, but with name that differs - personal info should remain the same.
Just like u've done it with Keanu Reeves